pysb/simulator/cuda_ssa.py

```diff
@@ -0,0 +1,254 @@
+from __future__ import print_function
+
+#
+try:
+    import pycuda
+    import pycuda.autoinit
+    import pycuda as cuda
+    import pycuda.compiler
+    import pycuda.tools as tools
+    import pycuda.driver as driver
+    import pycuda.gpuarray as gpuarray
+except ImportError:
+    pycuda = None
+
+import numpy as np
+import os
+import time
+from pysb.logging import setup_logger
+import logging
+from pysb.pathfinder import get_path
+from pysb.simulator.base import SimulationResult, SimulatorException
+from pysb.simulator.ssa_base import SSABase
+
+
+class CUDASimulator(SSABase):
+    """
+    CUDA simulator for NVIDIA gpus
+
+    Requires pycuda, cuda, and cuda compatible NVIDIA gpus.
```

Comment:
Repeated `cuda` here. Maybe provide a link to pycuda?

Reply:
Done